---
title: Long-read metagenome-assembled genomes improve identification of novel complete
  biosynthetic gene clusters in a complex microbial activated sludge ecosystem

# Authors
# A YAML list of author names
# If you created a profile for a user (e.g. the default `admin` user at `content/authors/admin/`), 
# write the username (folder name) here, and it will be replaced with their full name and linked to their profile.
authors:
- Roberto SÃ¡nchez-Navarro
- Matin Nuhamunada
- Omkar S Mohite
- Kenneth Wasmund
- Mads Albertsen
- Lone Gram
- Per H Nielsen
- Tilmann Weber
- Caitlin M Singleton

# Author notes (such as 'Equal Contribution')
# A YAML list of notes for each author in the above `authors` list
author_notes: []

date: '2022-12-20'

# Date to publish webpage (NOT necessarily Bibtex publication's date).
publishDate: '2025-09-03T04:32:37.817813Z'

# Publication type.
# A single CSL publication type but formatted as a YAML list (for Hugo requirements).
publication_types:
- article-journal

# Publication name and optional abbreviated publication name.
publication: '*American Society for Microbiology*'
publication_short: ''

doi: 10.1128/msystems.00632-22

abstract: Microorganisms produce a wide variety of secondary/specialized metabolites
  (SMs), the majority of which are yet to be discovered. These natural products play
  multiple roles in microbiomes and are important for microbial competition, communication,
  and success in the environment. SMs have been our major source of antibiotics and
  are used in a range of biotechnological applications. In silico mining for biosynthetic
  gene clusters (BGCs) encoding the production of SMs is commonly used to assess the
  genetic potential of organisms. However, as BGCs span tens to over 200 kb, identifying
  complete BGCs requires genome data that has minimal assembly gaps within the BGCs,
  a prerequisite that was previously only met by individually sequenced genomes. Here,
  we assess the performance of the currently available genome mining platform antiSMASH
  on 1,080 high-quality metagenome-assembled bacterial genomes (HQ MAGs) previously
  produced from wastewater treatment plants (WWTPs) using a combination of long-read
  (Oxford Nanopore) and short-read (Illumina) sequencing technologies. More than 4,200
  different BGCs were identified, with 88% of these being complete. Sequence similarity
  clustering of the BGCs implies that the majority of this biosynthetic potential
  likely encodes novel compounds, and few BGCs are shared between genera. We identify
  BGCs in abundant and functionally relevant genera in WWTPs, suggesting a role of
  secondary metabolism in this ecosystem. We find that the assembly of HQ MAGs using
  long-read sequencing is vital to explore the genetic potential for SM production
  among the uncultured members of microbial communities.

# Summary. An optional shortened abstract.
summary: ''

tags:
- activated sludge
- biosynthetic gene cluster
- metagenome-assembled genome
- secondary metabolite
- wastewater treatment plant

# Display this page in a list of Featured pages?
featured: false

# Links
url_pdf: ''
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Custom links (uncomment lines below)
# links:
# - name: Custom Link
#   url: http://example.org

# Publication image
# Add an image named `featured.jpg/png` to your page's folder then add a caption below.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects: ['internal-project']` links to `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []
links:
- name: URL
  url: http://dx.doi.org/10.1128/msystems.00632-22
---

Add the **full text** or **supplementary notes** for the publication here using Markdown formatting.
