---
title: 'BGCFlow: systematic pangenome workflow for the analysis of biosynthetic gene
  clusters across large genomic datasets'

# Authors
# A YAML list of author names
# If you created a profile for a user (e.g. the default `admin` user at `content/authors/admin/`), 
# write the username (folder name) here, and it will be replaced with their full name and linked to their profile.
authors:
- Matin Nuhamunada
- Omkar S Mohite
- Patrick V Phaneuf
- Bernhard O Palsson
- Tilmann Weber

# Author notes (such as 'Equal Contribution')
# A YAML list of notes for each author in the above `authors` list
author_notes: []

date: '2024-06-10'

# Date to publish webpage (NOT necessarily Bibtex publication's date).
publishDate: '2025-09-03T04:32:37.683803Z'

# Publication type.
# A single CSL publication type but formatted as a YAML list (for Hugo requirements).
publication_types:
- article-journal

# Publication name and optional abbreviated publication name.
publication: '*Oxford University Press (OUP)*'
publication_short: ''

doi: 10.1093/nar/gkae314

abstract: 'Genome mining is revolutionizing natural products discovery efforts. The
  rapid increase in available genomes demands comprehensive computational platforms
  to effectively extract biosynthetic knowledge encoded across bacterial pangenomes.
  Here, we present BGCFlow, a novel systematic workflow integrating analytics for
  large-scale genome mining of bacterial pangenomes. BGCFlow incorporates several
  genome analytics and mining tools grouped into five common stages of analysis such
  as: (i) data selection, (ii) functional annotation, (iii) phylogenetic analysis,
  (iv) genome mining, and (v) comparative analysis. Furthermore, BGCFlow provides
  easy configuration of different projects, parallel distribution, scheduled job monitoring,
  an interactive database to visualize tables, exploratory Jupyter Notebooks, and
  customized reports. Here, we demonstrate the application of BGCFlow by investigating
  the phylogenetic distribution of various biosynthetic gene clusters detected across
  42 genomes of the Saccharopolyspora genus, known to produce industrially important
  secondary/specialized metabolites. The BGCFlow-guided analysis predicted more accurate
  dereplication of BGCs and guided the targeted comparative analysis of selected RiPPs.
  The scalable, interoperable, adaptable, re-entrant, and reproducible nature of the
  BGCFlow will provide an effective novel way to extract the biosynthetic knowledge
  from the ever-growing genomic datasets of biotechnologically relevant bacterial
  species.'

# Summary. An optional shortened abstract.
summary: ''

tags: []

# Display this page in a list of Featured pages?
featured: false

# Links
url_pdf: ''
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Custom links (uncomment lines below)
# links:
# - name: Custom Link
#   url: http://example.org

# Publication image
# Add an image named `featured.jpg/png` to your page's folder then add a caption below.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects: ['internal-project']` links to `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []
links:
- name: URL
  url: http://dx.doi.org/10.1093/nar/gkae314
---

Add the **full text** or **supplementary notes** for the publication here using Markdown formatting.
