---
title: 'BGC Atlas: a web resource for exploring the global chemical diversity encoded
  in bacterial genomes'

# Authors
# A YAML list of author names
# If you created a profile for a user (e.g. the default `admin` user at `content/authors/admin/`), 
# write the username (folder name) here, and it will be replaced with their full name and linked to their profile.
authors:
- Caner Bağcı
- Matin Nuhamunada
- Hemant Goyat
- Casimir Ladanyi
- Ludek Sehnal
- Kai Blin
- Satria A Kautsar
- Azat Tagirdzhanov
- Alexey Gurevich
- Shrikant Mantri
- Christian von Mering
- Daniel Udwary
- Marnix H Medema
- Tilmann Weber
- Nadine Ziemert

# Author notes (such as 'Equal Contribution')
# A YAML list of notes for each author in the above `authors` list
author_notes: []

date: '2024-10-29'

# Date to publish webpage (NOT necessarily Bibtex publication's date).
publishDate: '2025-09-03T04:32:37.635781Z'

# Publication type.
# A single CSL publication type but formatted as a YAML list (for Hugo requirements).
publication_types:
- article-journal

# Publication name and optional abbreviated publication name.
publication: '*Oxford University Press (OUP)*'
publication_short: ''

doi: 10.1093/nar/gkae953

abstract: Secondary metabolites are compounds not essential for an organism's development,
  but provide significant ecological and physiological benefits. These compounds have
  applications in medicine, biotechnology and agriculture. Their production is encoded
  in biosynthetic gene clusters (BGCs), groups of genes collectively directing their
  biosynthesis. The advent of metagenomics has allowed researchers to study BGCs directly
  from environmental samples, identifying numerous previously unknown BGCs encoding
  unprecedented chemistry. Here, we present the BGC Atlas (https://bgc-atlas.cs.uni-tuebingen.de),
  a web resource that facilitates the exploration and analysis of BGC diversity in
  metagenomes. The BGC Atlas identifies and clusters BGCs from publicly available
  datasets, offering a centralized database and a web interface for metadata-aware
  exploration of BGCs and gene cluster families (GCFs). We analyzed over 35 000 datasets
  from MGnify, identifying nearly 1.8 million BGCs, which were clustered into GCFs.
  The analysis showed that ribosomally synthesized and post-translationally modified
  peptides are the most abundant compound class, with most GCFs exhibiting high environmental
  specificity. We believe that our tool will enable researchers to easily explore
  and analyze the BGC diversity in environmental samples, significantly enhancing
  our understanding of bacterial secondary metabolites, and promote the identification
  of ecological and evolutionary factors shaping the biosynthetic potential of microbial
  communities.

# Summary. An optional shortened abstract.
summary: ''

tags:
- metadata
- agriculture
- biotechnology
- genes
- genome
- bacterial peptides
- gene clusters
- metabolites
- biosynthesis
- metagenomics
- datasets

# Display this page in a list of Featured pages?
featured: false

# Links
url_pdf: ''
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Custom links (uncomment lines below)
# links:
# - name: Custom Link
#   url: http://example.org

# Publication image
# Add an image named `featured.jpg/png` to your page's folder then add a caption below.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects: ['internal-project']` links to `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []
links:
- name: URL
  url: https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkae953/7848841
---

Add the **full text** or **supplementary notes** for the publication here using Markdown formatting.
