---
title: Unlocking the biosynthetic potential of Paenibacilli through a genus-wide exploration
  of gene clusters for secondary metabolite production

# Authors
# A YAML list of author names
# If you created a profile for a user (e.g. the default `admin` user at `content/authors/admin/`), 
# write the username (folder name) here, and it will be replaced with their full name and linked to their profile.
authors:
- Lijie Song
- Matin Nuhamunada
- Tilmann Weber
- Ákos T Kovács

# Author notes (such as 'Equal Contribution')
# A YAML list of notes for each author in the above `authors` list
author_notes: []

date: '2025-01-24'

# Date to publish webpage (NOT necessarily Bibtex publication's date).
publishDate: '2025-09-03T04:32:37.543214Z'

# Publication type.
# A single CSL publication type but formatted as a YAML list (for Hugo requirements).
publication_types:
- article-journal

# Publication name and optional abbreviated publication name.
publication: ''
publication_short: ''

doi: 10.1101/2025.01.22.634348

abstract: The genus Paenibacillusis a prolific producer of secondary metabolites with
  diverse ecological and industrial applications. However, a comprehensive overview
  of the biosynthetic gene cluster (BGC) diversity and distribution throughout the
  genus has been limited. Here, we performed large-scale genome mining on 284 high-quality
  genomes and generated a non-redundant dataset of 126 representative genomes to explore
  the biosynthetic potential of this genus. A total of 3,273 BGCs were identified
  from the 284 genomes that clustered into 1,013 gene cluster families (GCFs), with
  98.7% classified as unknown, indicating vast potential for novel secondary metabolite
  discovery in thePaenibacillusgenus. Comparative analysis revealed significant phylogenetic
  and clade-specific distribution patterns of GCFs, with certain clades enriched in
  unique biosynthetic pathways while others exhibited low similarity to known BGCs,
  suggesting evolutionary adaptation to diverse ecological niches. This study uncovers
  the rich and largely untapped biosynthetic potential of the genusPaenibacillus,
  providing a foundation for future exploration of its natural products and their
  applications in biotechnology and medicine.IMPORTANCEBacterial secondary metabolites
  have been instrumental in the development of antibiotics, antifungals, and other
  bioactive compounds. The genusPaenibacillusis an underexplored source of such metabolites,
  with significant potential for novel discoveries. By integrating genome mining and
  phylogenetic analysis, this study systematically characterizes the diversity, distribution,
  and novelty of biosynthetic gene cluster across the genus. The identification of
  clade-specific biosynthetic patterns and numerous unknown gene cluster families
  highlightsPaenibacillusas a promising target for uncovering novel compounds with
  ecological and therapeutic relevance. These findings not only expand our understanding
  of bacterial secondary metabolite biosynthesis but also offer new opportunities
  for the development of sustainable biotechnological applications.

# Summary. An optional shortened abstract.
summary: ''

tags:
- Paenibacilli
- natural products
- genome mining

# Display this page in a list of Featured pages?
featured: false

# Links
url_pdf: ''
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Custom links (uncomment lines below)
# links:
# - name: Custom Link
#   url: http://example.org

# Publication image
# Add an image named `featured.jpg/png` to your page's folder then add a caption below.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects: ['internal-project']` links to `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []
links:
- name: URL
  url: https://www.biorxiv.org/content/10.1101/2025.01.22.634348v1.abstract
---

Add the **full text** or **supplementary notes** for the publication here using Markdown formatting.
